### ISSUE TITLE
Still working ?

### ISSUE THREAD

**Issue by romain325** [thm-community/thm-api-py#1]
THM seems to have changed their API, will it be possible to use the API in the future ?

**Comment by szymex73** [comment]
Yes, I know about the changes and I'm planning on updating the wrapper to reflect those but at this moment I don't have enough time to do so.

Feel free to contribute those if you feel so though :)

**Comment by romain325** [comment]
Np !
And i'll be happy to contribute, the problem is that I can't manage to find any informations about the new endpoints

**Comment by AlanBlanchet** [comment]
I tried accessing the API with php and wasn't able because of the Cloudflare instance that they have...
Is this wrapper able to get data event with Cloudflare ?

Because at the moment nobody is actually able to access the API from script right ? If not how can we access it ?

**Comment by szymex73** [comment]
> I tried accessing the API with php and wasn't able because of the Cloudflare instance that they have...
> Is this wrapper able to get data event with Cloudflare ?
> 
> Because at the moment nobody is actually able to access the API from script right ? If not how can we access it ?

At the moment because of Cloudflare being in front of the website the wrapper sometimes works and sometimes doesn't, same thing will happen if you make a wrapper of your own. If Cloudflare deems your requests suspicious or whatev then you'll be out of luck.

It's out of my control and even if the wrapper doesn't work at the moment, I'd like to keep the thing at least partially up-to-date as it's one of the few resources that map the current(/past) API routes.

**Comment by GnarLito** [comment]
If u are planning to keep the API lib up to date, use this postman documentation I wrote https://documenter.getpostman.com/view/18269560/UVCB9j5e
